packages/core/core/src/Environment.js

```diff
@@ -84,10 +86,12 @@ export function createEnvironment({
     engines,
     includeNodeModules,
     outputFormat,
+    sourceType: sourceType ?? 'module',
```

Comment:
(along with above, assuming `null` is a desirable type for `SourceMap`)

```suggestion
    sourceType,
```